flight/Modules/Battery/battery.c

```diff
@@ -185,6 +185,8 @@ static void batteryTask(void * parameters)
 				AlarmsClear(SYSTEMALARMS_ALARM_BATTERY);
 		} else {
 			flightBatteryData.Voltage = 0;
+			AlarmsSet(SYSTEMALARMS_ALARM_BATTERY, SYSTEMALARMS_ALARM_ERROR);
+			AlarmsSet(SYSTEMALARMS_ALARM_FLIGHTTIME, SYSTEMALARMS_ALARM_ERROR);
```

Comment:
Setting the alarm here means you are now required to have BOTH a current sensor and voltage sensor to use the battery module without error. It's also out of the scope of this PR. This should be reverted IMO.